src/components/organisms/NewOrderBookWidget/NewOrderBookWidget.test.tsx

```diff
@@ -373,17 +373,10 @@ describe.skip('NewOrderBookWidget Component', () => {
         });
     });
 
-    // describe('Variants', () => {
-    //     it('should display the current market price in the correct color', () => {
```

Comment:
This test was removed because in NewOrderBookWidget, the current market price color is neutral-50 irregardless of whether it's Itayose.